source/config/featureFlagEnums.py

```diff
@@ -0,0 +1,49 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2022 NV Access Limited
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+
+"""
+Feature flag value enumerations.
+Some feature flags require feature specific options, this file defines those options.
+All feature flags enums must have a 'DEFAULT'
+"""
+import enum
+import logging
+import sys
+import typing
+
+from typing_extensions import Protocol  # Python 3.8 adds native support
+
+
+class FeatureFlagEnumProtocol(Protocol):

```

Comment:
Could these use `DisplayStringEnum` as a base well, to ensure each item has a display string label?
This can replace the `translatedOptions` injection.